sdks/python/apache_beam/internal/gcp/auth.py

```diff
@@ -123,6 +123,7 @@ def get_service_credentials():
       logging.debug('Connecting using Google Application Default '
                     'Credentials.')
       return credentials
-    except Exception:
-      logging.warning('Unable to find default credentials to use.')
-      raise
+    except Exception as e:
+      logging.warning(
+          'Unable to find default credentials to use: %s\n'
+          'Connecting anonymously.' % e)
```

Comment:
<!--new_thread; commit:ae90401b8ff353640d5d31084b3b9bfd4a396215; resolved:0-->
Should we return an explicit `None` here?